modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -4369,6 +4369,12 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description></description>
```

Comment:
@carloslancha, @markocikos I wasn't sure this was needed but since [other](https://github.com/liferay/liferay-portal/blob/2ba32f018d39b3eaacd32ff6c0fed7cfb806cff7/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld#L307-L312) [tags](https://github.com/liferay/liferay-portal/blob/2ba32f018d39b3eaacd32ff6c0fed7cfb806cff7/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld#L1042-L1047) have it I though it wouldn't hurt.

Reply:
Maybe it also needs 

```xml
<attribute>
    <description></description>
    <name>propsTransformerServletContext</name>
    <required>false</required>
    <rtexprvalue>true</rtexprvalue>
</attribute>
```

but I'll wait for your comments

Reply:
Agree on adding `propsTransformer`.

Why would we add `propsTransformerServletContext`? Are we using it somewhere but it falls under dynamic attributes, or is it commonly used in other tags that also have `propsTransformer`?

Reply:
Yes, other tags that have the `propsTransformer` attribute usually have `propsTransformerServletContext`, so I just wanted to make that identical.

Reply:
No idea what is `propsTransformerServletContext` for. I'd say no to add stuff until we need it, but in this case I think we need some context. @jbalsas please?

Reply:
@carloslancha I know what it's for :smile:

Reply:
@carloslancha see [here](https://github.com/liferay/liferay-portal/blob/9d49616b01cc7f892e544d79db90fcc42775c08b/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java#L319-L323):

What I understand is that if a `propsTransformerServletContext` is passed, it will be used to resolve (load) the `propsTransformer` js module.

>  I'd say no to add stuff until we need it, 

Since `ButtonTag` and `DropdownMenuTag` have both `propsTransformer` and `propsTransformerServletContext` as attributes, I'm just following the same convention and for me it makes sense